lib/mrkt/concerns/custom_objects.rb

```diff
@@ -0,0 +1,54 @@
+module Mrkt
+  module CustomObjects
+    def get_list_of_custom_objects(object_names)
+      params = {
+        names: object_names.join(',')
+      }
+
+      get('/rest/v1/customobjects.json', params)
+    end
+
+    def describe_custom_object(object_name)
+      fail Mrkt::Errors::Unknown if !object_name || object_name.empty?
+
+      get("/rest/v1/customobjects/#{object_name}/describe.json")
+    end
+
+    def createupdate_custom_objects(object_name, objects, action: 'createOrUpdate', dedupe_by: 'dedupeFields')
```

Comment:
It should be called `name` and `input` as in the [documentation](http://developers.marketo.com/documentation/custom-api/createupdateupsert-custom-objects/).